### ISSUE TITLE
In active sync, delete option is missing from the dropdown of API's

### ISSUE THREAD

**Issue by sanilnayak** [sparrowapp-dev/sparrow-app#672]
### **🐛 Bug Report**

In active sync, delete option is missing from the dropdown of API's

**Steps to reproduce:**

- Login to sparrow and import a active sync collection from swagger
- Click on 3 dots of API to open option box 

**Expected Behaviour:** Delete option should be there 
<img width="578" alt="Screenshot 2024-03-21 at 3 59 37 PM" src="https://github.com/sparrowapp-dev/sparrow-app/assets/129532962/6f2910dd-9edf-4281-97f0-a808e0b623f7">

**Actual Behaviour:** Delete option is missing 
<img width="523" alt="Screenshot 2024-03-21 at 4 00 42 PM" src="https://github.com/sparrowapp-dev/sparrow-app/assets/129532962/1597867f-2adc-4d34-9d79-30a28d0d0a25">

**Comment by LordNayan** [comment]
This is correct behaviour active synced apis cannot be deleted or modified. @ayushadhikari93

**Comment by jshalini26** [comment]
Working in windows
@sanilnayak Please test

**Comment by shreyas070707** [comment]
currently not able to test, so moving it to backlog
cc- @ayushadhikari93 @sanilnayak @jshalini26 @nagesh953
